src/forms-opportunities/handler.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import RUMAPIClient from '@adobe/spacecat-shared-rum-api-client';
+import { getRUMDomainkey } from '../support/utils.js';
+import { AuditBuilder } from '../common/audit-builder.js';
+import { wwwUrlResolver } from '../common/audit.js';
+
+const DAILY_THRESHOLD = 200;
+const INTERVAL = 7; // days
+const FORMS_OPPTY_QUERIES = [
+  'cwv',
+  'form-vitals',
+];
+
+export async function formsAuditRunner(auditUrl, context, site) {
+  const rumAPIClient = RUMAPIClient.createFrom(context);
+  const domainkey = await getRUMDomainkey(site.getBaseURL(), context);
+  const options = {
+    domain: auditUrl,
+    domainkey,
+    interval: INTERVAL,
+    granularity: 'hourly',
+  };
+
+  const formsAuditLinks = await rumAPIClient.queryMulti(FORMS_OPPTY_QUERIES, options);
```

Comment:
cc'ing @naydav for transparency: form-vitals audit starting to use cwv query as of this PR. Let's not forget to adjust form-vitals query when the planned response body changes are made to the cwv query

Reply:
@ekremney you're referring to https://github.com/adobe/spacecat-api-service/pull/574 right?
Good point, @05rahulsingh please monitor this PR, once merged, you may need to update your audit

Reply:
@05rahulsingh since the query is now multi, its better to name `formsAuditLinks` something generic like `queryResults` or so